MapboxDirections/MBDirections.swift

```diff
@@ -141,10 +141,17 @@ open class Directions: NSObject {
         
         self.accessToken = accessToken!
         
+        if(host == nil || host!.isEmpty){
+            let apiBaseURLString = Bundle.main.object(forInfoDictionaryKey: "MGLMapboxAPIBaseURL") as? String
+            self.apiEndpoint = apiBaseURLString != nil ? URL(string: apiBaseURLString!)!: URL(string: "api.mapbox.com")!
```

Comment:
As long as `host` is set, the existing code below already sets `apiEndpoint`, so there’s no need to construct a URL here.

`URL(string:)` takes an absolute URL, not just a hostname. The builds are failing because `api.mapbox.com` isn’t a full URL. For example:

```
V4Tests
    ✗ testGeoJSON, XCTAssertNil failed: "Error Domain=NSURLErrorDomain Code=-1002 "unsupported URL" UserInfo={NSErrorFailingURLStringKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSErrorFailingURLKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSUnderlyingError=0x7f8f4afae030 {Error Domain=NSURLErrorDomain Code=-1002 "unsupported URL" UserInfo={NSUnderlyingError=0x7f8f4aeb0750 {Error Domain=kCFErrorDomainCFNetwork Code=-1002 "(null)"}, NSErrorFailingURLStringKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSErrorFailingURLKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSLocalizedDescription=unsupported URL}}, NSLocalizedDescription=unsupported URL}" - Error: Error Domain=NSURLErrorDomain Code=-1002 "unsupported URL" UserInfo={NSErrorFailingURLStringKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSErrorFailingURLKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSUnderlyingError=0x7f8f4afae030 {Error Domain=NSURLErrorDomain Code=-1002 "unsupported URL" UserInfo={NSUnderlyingError=0x7f8f4aeb0750 {Error Domain=kCFErrorDomainCFNetwork Code=-1002 "(null)"}, NSErrorFailingURLStringKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSErrorFailingURLKey=//v4/directions/[REDACTED].driving/-122.42,37.78%3B-77.03,38.91.json?alternatives=[REDACTED]&instructions=text&geometry=geojson&steps=[REDACTED]&access_token=pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede, NSLocalizedDescription=unsupported URL}}, NSLocalizedDescription=unsupported URL}
    ✗ testGeoJSON, XCTAssertNotNil failed -
```